editoast/migrations/2025-09-30-085502_etcs_level2_from_etcs_brake_params/down.sql

```diff
@@ -0,0 +1,7 @@
+-- This file should undo anything in `up.sql`
+
+UPDATE rolling_stock
+SET supported_signaling_systems = supported_signaling_systems || ARRAY['ETCS_LEVEL2']
```

Comment:
nit (more expressive IMO):
```suggestion
SET supported_signaling_systems = array_append(supported_signaling_systems, 'ETCS_LEVEL2')
```